### PR TITLE
PowerNap 

### PR DESCRIPTION
**Pull request by arncv**
Powernap Draft 1

### CODE REVIEW

File: PowerNap/blog.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
+# Boost Your Productivity with PowerNap - A Time Tracker Web App Integrated with Courier API for Effortless Notification Management.
+
+## Background
+
+PowerNap is a simple and elegant time tracker web application that allows users to set up a timer for their power nap sessions. The application was built to help individuals improve their productivity by incorporating short naps into their work routine.For this project, we utilized Node.js and Express.js for the backend and vanilla JS, HTML and CSS for frontend development respectively. We also integrated Courier API to enable the app to send push notifications to users.
```
Comment (shreythecray) [inline PowerNap/blog.md:5]: Link "Courier API" to https://courier.com/

File: PowerNap/blog.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+
+I chose to use Node.js for its efficient and lightweight nature, which makes it perfect for building scalable applications.For the frontend, I chose to use vanilla JavaScript because it's lightweight, fast, and compatible with all major browsers along with CSS Finally, I incorporated the Courier API to send notifications to users in real-time, which enhances the user experience and improves productivity.
```
Comment (shreythecray) [inline PowerNap/blog.md:7]: Great introduction! In the first paragraph you refer to yourself as "we" and in the second one you use "I". I'd recommend sticking to one person, so either write as "I did this" or "we did this".

File: PowerNap/blog.md [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+   ```
+   We will also create a function playSound that will play a sound when the timer reaches zero.
+   ```javascript
+   function playSound() {
+   audio.play();
+   }
+   ```
+   Next, we will create a function updateTimer that will take care of updating the timer. We will use the setInterval method to update the clock every second.
+
+   ```javascript
```
Comment (shreythecray) [inline PowerNap/blog.md:78]: Thanks for breaking down the code!

File: PowerNap/blog.md [hunk 4]
```diff
@@ -0,0 +1,350 @@
+# Boost Your Productivity with PowerNap - A Time Tracker Web App Integrated with Courier API for Effortless Notification Management.
```
Comment (shreythecray) [inline PowerNap/blog.md:1]: This file should be titled readme.md

File: PowerNap/blog.md [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+
+   And the basic functionality of the timer is complete! You can check out the final code [here](https://github.com/arncv/PowerNap/blob/main/public/script.js).
+
+3. **Styling the timer using CSS**
+   
+   ![PowerNap](https://i.imgur.com/O4njIL2_d.webp?maxwidth=1520&fidelity=grand)
+
+
+   Using CSS, we can build beautiful and functional UI design for a Pomodoro timer and a music player. The design features a calming blue gradient background, and the typography is set in the Lato font family.
+   PowerNap includes a menu with buttons for setting the timer length and selecting the type of session, as well as a progress bar that displays the remaining time. The timer is contained within a circular container, and the countdown clock is prominently displayed in the center.
```
Comment (shreythecray) [inline PowerNap/blog.md:126]: Add new line

**Review by shreythecray — CHANGES_REQUESTED** [review]
This blog looks great but there are some structural changes that need to be made (to bring back the deleted files). I've also added some comments to the blog directly.

**Comment by arncv** [comment]
I've made the changes requested in another Pull Request and also rectified some of the structural errors of this Pull Request

Link to PR - https://github.com/trycourier/community-projects/pull/17

**Pull request closed without merging**